app/assets/stylesheets/application.scss

```diff
@@ -19,6 +19,7 @@
 @import 'atoms/form-widths';
 @import 'atoms/card';
 @import 'atoms/step-anchor';
+@import 'atoms/emoji';
```

Comment:
Unfortunately css can get completely screwy if you re-order rules/styles so I'm hesitant to re-order things too too much. Everything above is *not* alpha so I don't know if I want to make that change and untangle that. That cool, @jessieay?